documentation/domains/Domain.md

```diff
@@ -233,12 +222,16 @@ The current status of the operation of the WebLogic domain. Updated automaticall
 
 | Name | Type | Description |
 | --- | --- | --- |
+| `auxiliaryImages` | Array of [Auxiliary Image](#auxiliary-image) | Optionally use auxiliary images to provide Model in Image model, application archive and WebLogic Deploy Tooling files. This is a useful alternative for providing these files without requiring modifications to the pod's base image 'domain.spec.image'. This feature internally uses a Kubernetes emptyDir volume and Kubernetes init containers to share the files from the additional images with the pod. |
+| `auxiliaryImageVolumeMedium` | string | The emptyDir volume medium. This is an advanced setting that rarely needs to be configured. Defaults to unset, which means the volume's files are stored on the local node's file system for the life of the pod. |
+| `auxiliaryImageVolumeMountPath` | string | The auxiliary image volume mount path. This is an advanced setting that rarely needs to be configured. Defaults to '/aux', which means the emptyDir volume will be mounted at '/aux' path in the WebLogic-Server container within every pod. The defaults for 'modelHome' and 'wdtInstallHome' will start with the new mount path, and files from 'sourceModelHome' and 'sourceWDTInstallHome' will be copied to the new default locations. |
```

Comment:
'/aux' -> Please fix all of the following by applying code font; use back ticks instead of straight quotes.
'modelHome' 
'wdtInstallHome'
'sourceWDTInstallHome'
'sourceModelHome' and 'sourceWDTInstallHome'